static/image/refresh-disabled.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
```

Comment:
You can create a disabled version of the `refresh.svg` file by creating a copy of it and adding a `fill` attribute with value `#dedede` to the first `path` element in this new copy.

Reply:
And I used photoshop to generate an entirely new file.. 😩